website/i18n/zh/docusaurus-plugin-content-blog/2022/01/11/building-apisix-in-ubuntu-for-arm.md

```diff
@@ -9,34 +9,35 @@ keywords:
 - Apache APISIX
 - arm
 - ubuntu
-- m1
-description: 本文为大家描述了在 M1 中启用 arm 版 ubuntu 通过源码构建 Apache APISIX 的详细操作步骤，通过阅读本文，大家对于在 arm ubuntu 上构建 Apache APISIX 更加快捷。
+- Apple Macbook Pro M1
+description: 笔者使用的是 M1 芯片的 Macbook Pro，通过阅读本文，您将了解如何在 ARM Ubuntu 中通过源码构建 Apache APISIX。
 tags: [Technology]
 ---
 
-> 本篇文章记录了通过源码在 arm 版 ubuntu 中构建 Apache APISIX 的详细踩坑记录，ubuntu 环境使用的是 [https://multipass.run/](https://multipass.run/) 中的 Multipass 。因为它可以在 M1 上使用～
+> 笔者使用的是 M1 芯片的 Macbook Pro，借助 [https://multipass.run/](https://multipass.run/) 安装了 Ubuntu 系统，本文便记录了如何在此环境中通过源码构建 Apache APISIX。
 
 <!--truncate-->
 
 ## 克隆源码
 
-首先根据 [官方文档](https://apisix.apache.org/zh/docs/apisix/how-to-build#%E9%80%9A%E8%BF%87%E6%BA%90%E7%A0%81%E5%8C%85%E5%AE%89%E8%A3%85) 安装 APISIX 源码仓库。并且进入项目文件。
+首先根据 [官方文档](https://apisix.apache.org/zh/docs/apisix/how-to-build#%E9%80%9A%E8%BF%87%E6%BA%90%E7%A0%81%E5%8C%85%E5%AE%89%E8%A3%85) 克隆 APISIX 源码仓库，然后进入项目目录。
 
 ```shell
-cd apisix-2.11.0
+git clone https://github.com/apache/apisix.git
```

Comment:
This will use the master branch code, you need to checkout to your correct version.